native/templates/android/template/app/cocos_engine.gradle

```diff
@@ -0,0 +1,11 @@
+
+android {
```

Comment:
native/templates is moved to cocos-engine/templates. And why add this file?

Reply:
game-activity develop earlier than last merged pr. I have fixed it